### ISSUE TITLE
adding lake association data to database

### ISSUE THREAD

**Issue by joneslabND** [MFEh2o/db#11]
eventually we'll want to put this stuff in the database; although we may have to leave some data out due to IRB sorts of stuff

**Comment by joneslabND** [comment]
I think @ctsolomon knows more about this

**Comment by ctsolomon** [comment]
Yes - I just put it back on my active to-do list at the beginning of this week, as it turns out. I need to start a conversation with Marco and Dane about it - I'll do that.
